onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -83,28 +83,26 @@ std::ostream& operator<<(std::ostream& out, std::pair<const SequentialExecutionP
   const SessionState& session_state = *planinfo.second;
 
   const auto& name_idx_map = session_state.GetOrtValueNameIdxMap();
-  InlinedHashMap<int, std::string_view> index_to_name;
-  index_to_name.reserve(name_idx_map.Size());
+  std::map<int, std::string_view> index_to_name;  // order by Node_Arg index by default
```

Comment:
why we change back to std map?

Reply:
In order to print the Node_Arg info sorted by node index. InlinedHashMap is unordered map so it will print Node_Arg info randomly. 
This function is used only under debugging purposes so I assume perf is not a concern.